seurat_examples/seurat_load_example.R

```diff
@@ -0,0 +1,47 @@
+## Install necessary R packages (should only need to be done once) - uncomment to run
+#install.packages("tidyverse")
+#if (!requireNamespace("BiocManager", quietly = TRUE))
+#    install.packages("BiocManager")
+#BiocManager::install(version = "3.10")
+#install.packages('Seurat')
+#install.packages('patchwork')
```

Comment:
Not necessary, but I've used code like this (https://stackoverflow.com/a/4090208/63093):
```R
list.of.packages <- c("ggplot2", "Rcpp")
new.packages <- list.of.packages[!(list.of.packages %in% installed.packages()[,"Package"])]
if(length(new.packages)) install.packages(new.packages)
```
Or sometimes this (https://stackoverflow.com/a/19873732/63093):
```R
if (!require("pacman")) install.packages("pacman")
pacman::p_load(package1, package2, package_n)
```

Reply:
Do you think `BiocManager::install(version = "3.10")` is necessary? If so, can I put it after something like you suggest, or would it be necessary to do that before the Seurat/patchwork install?

Reply:
I don't think you absolutely have to specify a specific version here, but that's up to you. I don't think base R allows that (via `install.packages`), so you'd need to use BiocManager to do it... Maybe something like:
```R
if (!requireNamespace("BiocManager", quietly = TRUE))
    install.packages("BiocManager")
BiocManager::install(version = "3.10")
if (!require("pacman")) install.packages("pacman")
pacman::p_load(Seurat, patchwork)
```

If you don't need that specific version, this will work (it tries using BiocManager if the package isn't on CRAN):
```R
if (!require("pacman")) install.packages("pacman")
pacman::p_load(Seurat, patchwork)
```